config/locales/en.yml

```diff
@@ -33,3 +33,7 @@ en:
   api:
     errors:
       invalid_body: Please submit proper sign up data in request body.
+      confirmation_email: A confirmation email was sent to your account
```

Comment:
This doesn't look like an error, maybe this is not the best place for it?

Reply:
This message actually comes when a user tries to login when is not confirmed yet. This message also appears in the response error key.